src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs

```diff
@@ -73,7 +73,7 @@ public override ICommandLineParserResult ParseArgs(string[] args)
                 .Callback(f => ListIncludedFiles = f);
             Parser
                 .Setup<bool>("nozip")
-                .WithDescription("Publish in Run-From-Zip package. Requires the app to have AzureWebJobsStorage setting defined.")
+                .WithDescription("Turns the default Run-From-Zip mode off")
```

Comment:
nit: period

Question (perhaps outside of the scope of this change), but should the verbiage/flag here be `run-from-package` ?

Reply:
yes I think it should be updated to say `Run-From-Package`

Reply:
Sounds good. So, the flag would be `nopackage`? Or just the description and internal variable? Because if we change the flag to `--nopackage`, should we still continue to support `--nozip` (with a message) for a bit to not break?

Reply:
I'd just change the description. Leave the flag as is because otherwise like you said we will have to support both.

Reply:
Updated just the description